@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -54,10 +54,54 @@
 		>
 			<fieldset
 				class="tw-w-full tw-flex"
+				:class="{'tw-flex-col tw-gap-1': showPresetAmounts}"
 				:disabled="isAdding"
 				data-testid="bp-lend-cta-select-and-button"
 			>
-				<div class="amountDropdownWrapper">
+				<div
+					v-if="showPresetAmounts"
+					class="tw-flex tw-gap-1"
+				>
+					<template v-if="!showUnreservedAmountOnly">
+						<kv-ui-button
+							v-for="option in defaultAmountOptions"
+							:key="option"
+							:state="`${ unreservedAmount < option ? 'disabled' : ''}`"
```

Comment:
Could we add one more story for this logic, when the amount left is less than $75, but a increment of $25. So unreserved of $25 would be good, so we can see the rest of the buttons disabled.